match.py

```diff
@@ -0,0 +1,5 @@
+import requests 
+
+r = requests.get("https://github.com", verify=False)
```

Comment:
[DS130821: Disabled certificate validation](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS114352.md)
Extend default certificate validation, but do not disable or override default rules.

*Replacement:* Always use a valid certificate, even during testing.